### ISSUE TITLE
Training configuration

### ISSUE THREAD

**Issue by xiweix** [cvlab-columbia/pix2gestalt#39]
Hi, thank you for the wonderful work!
I want to ask if "**configs/sd-finetune-pix2gestalt-c_concat-256.yaml**" provided in this repo is the configuration file for finetuning pix2gestalt from the stable diffusion?
If so, would the training be simply done by running the below command?
          python main.py \
              -t \
              --base configs/sd-finetune-pix2gestalt-c_concat-256.yaml \
              --gpus 0,1,2,3,4,5,6,7 \
              --scale_lr False \
              --num_nodes 1 \
              --seed 42 \
              --check_val_every_n_epoch 2 \
              --finetune_from ckpt/sd-image-conditioned-v2.ckpt
Thanks!

**Comment by egeozguroglu** [comment]
Yes, it's the correct configuration if you're fine-tuning on our dataset.
